<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagsCompatibilityOptionsPage</class>
 <widget class="QWidget" name="TagsCompatibilityOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>705</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="wave_files">
     <property name="title">
      <string>WAVE files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Picard will tag WAVE files using ID3v2 tags. This is not supported by all software. For compatibility with software which does not support ID3v2 tags in WAVE files additional RIFF INFO tags can be written to the files. RIFF INFO has only limited support for tags and character encodings.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="write_wave_riff_info">
        <property name="text">
         <string>Also include RIFF INFO tags in the files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remove_wave_riff_info">
        <property name="text">
         <string>Remove existing RIFF INFO tags from WAVE files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="wave_riff_info_encoding">
        <property name="title">
         <string>RIFF INFO Text Encoding</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="wave_riff_info_enc_cp1252">
           <property name="text">
            <string>Windows-1252</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="wave_riff_info_enc_utf8">
           <property name="text">
            <string>UTF-8</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>274</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>write_wave_riff_info</sender>
   <signal>toggled(bool)</signal>
   <receiver>remove_wave_riff_info</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
